<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>100</y>
     <width>91</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Carsell.bild</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Kaufen</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QComboBox" name="comboPreis"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Farbe">
       <property name="text">
        <string>      Farbe</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="Modell">
       <property name="text">
        <string>    Modell</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Mark">
       <property name="text">
        <string>   Marke</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboModel"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboColor"/>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Suche">
       <property name="text">
        <string>Suche</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="Preis">
       <property name="text">
        <string>Preis bis</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboMark"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="Kraftstoff_2">
       <property name="text">
        <string>    Kraftstoff</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Verkaufen</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>576</x>
       <y>40</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>    Carsell.bild</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Spezifikationen eingeben</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineModell">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>     Marke</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="text">
            <string>    Modell</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="text">
            <string>   Farbe</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="text">
            <string>   Preis</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="erstellen">
           <property name="text">
            <string>Inserat erstellen</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineFarbe">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lineMarke">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="Kraftstoff">
           <property name="text">
            <string>Kraftstoff</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="comboKraft"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="linePreis">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>About</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
